datarepo-clienttests/src/main/java/scripts/testscripts/RetrieveSnapshot.java

```diff
@@ -18,15 +19,22 @@
 import java.nio.charset.StandardCharsets;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Optional;
+import org.apache.commons.lang3.StringUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import runner.config.TestUserSpecification;
 import scripts.testscripts.baseclasses.SimpleDataset;
 import scripts.utils.DataRepoUtils;
+import scripts.utils.SAMUtils;
 
 public class RetrieveSnapshot extends SimpleDataset {
   private static final Logger logger = LoggerFactory.getLogger(RetrieveSnapshot.class);
 
+  // Specify the name of a profile to use for the test with the TEST_RUNNER_BILLING_PROFILE_NAME
+  // env variable.
+  private static final String profileNameRaw = System.getenv("TEST_RUNNER_BILLING_PROFILE_NAME");
```

Comment:
java static final should be uppercase
```suggestion
  private static final String PROFILE_NAME_RAW = System.getenv("TEST_RUNNER_BILLING_PROFILE_NAME");
```

Reply:
Gah, you're right.  I got used to looking at the logger variables everywhere.  We should upper case those at some point

Reply:
I think `logger` is a ok exception to the uppercase rule.